<ui version="4.0" >
 <class>PagePlaylist</class>
 <widget class="QWidget" name="PagePlaylist" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Please add the movies you want to be included in your DVD. The movies will be played in order. You can reorder them by dragging. If you are going to include a menu, here you can also the set each one a title.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="addmovies_button" >
       <property name="text" >
        <string>Add Movie(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="removemovies_button" >
       <property name="text" >
        <string>Remove All Movies</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView" >
     <property name="acceptDrops" >
      <bool>true</bool>
     </property>
     <property name="dragEnabled" >
      <bool>true</bool>
     </property>
     <property name="dragDropMode" >
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
